LIMS_IMAGE/web/shopping/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.contrib.auth.models import User
+from accounts.models import *
+from orders.models import Order, Package
+from laboratory.models import Test, Location
+from datetime import datetime
+
+# Create your models here.
+class ShoppingCart(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(default=datetime.now)
+
+class CartItem(models.Model):
+    sample_type = models.CharField(max_length=100)
+    order_number = models.ForeignKey(Order, on_delete=models.CASCADE)
+    package = models.ForeignKey(Package, on_delete=models.CASCADE)
+
+    price = models.FloatField(blank=True)
+    cart = models.ForeignKey('ShoppingCart', on_delete=models.CASCADE)
+
+    TAX = 0.12
+
+    def price_with_tax(self):
+        TAX = 0.12

```

Comment:
you have declared the same constant twice

Reply:
If price_with_tax() is called from a CartItem object, I believe you could do self.TAX. Regardless, I think we should just remove the tax stuff for now.